### DISCUSSION TITLE
Custom Client Generator

### DISCUSSION THREAD

**Discussion by VeryAnonymousMan** [rustdesk/rustdesk#6507] [Q&A] [answered]
We are currently considering which plan to buy,
the one thing we would really like is custom client generation.
It says under Business that this feature is coming soon, but how long has it been in coming soon and is there an ETA for when it may be released?
ty

**Comment by rustdesk** [comment]
Currently we are struggling on this, https://github.com/rustdesk/rustdesk/discussions/6467. 
Today, we received 3 requests (including you) about this. We will put this in highest prioritiy.

**Reply by VeryAnonymousMan** [reply]
thank you for the quick reply
and best of luck to you guys, thank you for your hard work

**Reply by perplexityjeff** [reply]
Thank you as well for your hard work, me and my team will be watching for this feature to drop!

**Reply by ToniSchaad** [reply]
@rustdesk  i will put us also in the line of Pro-Plan-Users with Intrest of "Custom Client Generation". ATM i do it self-compiled on Windows and with configscript on Mac

**Reply by rustdesk** [reply]
Could you share how do you use configscript on Mac with RustDesk?

**Reply by ToniSchaad** [reply]
i put your Script (https://rustdesk.com/docs/en/self-host/client-deployment/#macos-bash) in a .sh-File and generate an App with it with Platypus. Works beside the "unverified developer"-Warning realy good on Intel- and M-based Macs

**Reply by rustdesk** [reply]
Thanks

**Reply by ToniSchaad** [reply]
you're welcome

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/6668#discussioncomment-8539983
